walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/MeetingRequest.java

```diff
@@ -21,7 +21,6 @@
 import java.util.HashSet;
 
 public final class MeetingRequest {
-
```

Comment:
Why was this file modified?

Reply:
I ran clang format, so it removed the extra white line.